### PR TITLE
Added my HyperViz project

### PR DESCRIPTION
**Pull request by F1nnM**
I hope, it's eligible, if not, what do I need to change?
Thank you for this awesome tool!

### CODE REVIEW

File: _data/projects/hyperviz.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: Analysing hyper-parameters made easy.
+site: https://github.com/F1nnM/HyperViz
+tags:
+  - python
+  - streamlit
+  - web
+  - machine-learning
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/F1nnM/HyperViz/issues
```
Comment (ritwik12) [inline _data/projects/hyperviz.yml]: @F1nnM Please use the link in the form of https://github.com/F1nnM/HyperViz/labels/good%20first%20issue.

File: _data/projects/hyperviz.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: HyperViz
+desc: Analysing hyper-parameters made easy.
+site: https://github.com/F1nnM/HyperViz
+tags:
+  - python
+  - streamlit
+  - web
+  - machine-learning
+upforgrabs:
+  name: hacktoberfest
```
Comment (ritwik12) [inline _data/projects/hyperviz.yml]: @F1nnM hacktoberfest will go away, want to rename it to something general such as `good first issue`? We can keep it to hacktoberfest if you want.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/hyperviz.yml (hunk 2 above)
Comment (F1nnM) [reply _data/projects/hyperviz.yml]: Ah yes. Changed it.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@F1nnM Thanks!!, Glad you liked the website.

**Pull request merged**